specification/loadtestservice/resource-manager/Microsoft.LoadTestService.Draft/stable/2022-12-01/loadtestservice.json

```diff
@@ -0,0 +1,1021 @@
+{
```

Comment:
Is there a view to compare this json against previous json

Reply:
When we create PR there are pipeline which generate API view, but for data plane I was not able to use it. Discussed with Mike also in our initial meeting where he was also not able to use it.

Reply:
Will check it again if we can use it somehow.